website/src/components/CustomerTestimonials/index.tsx

```diff
@@ -3,9 +3,11 @@
 import { useRef } from "react";
 import Image from "next/image";
 import Link from "next/link";
+import { FaQuoteLeft } from "react-icons/fa";
 import { HiArrowLeft, HiArrowRight } from "react-icons/hi2";
 import { FaHeart } from "react-icons/fa";
 import { manrope } from "@/lib/fonts";
+import ActionLink from "../ActionLink";
```

Comment:
Full path will make this a little more robust and help with project-wide search replace.

```suggestion
import ActionLink from "@/components/ActionLink";
```